papers/TIARA/azure-pipelines.yml

```diff
@@ -0,0 +1,69 @@
+# Python package
```

Comment:
This file should not be part of the release.

Reply:
I will remove this file.